gravitee-apim-console-webui/src/shared/components/gio-select-search/gio-select-search.component.scss

```
@@ -0,0 +1,62 @@
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+
+:host {
+  display: block;
+}
+
+.gio-select-search {
+  &__trigger {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 8px 12px;
+    border: 1px solid mat.m2-get-color-from-palette(gio.$mat-smoke-palette, lighter20);
+    border-radius: 4px;
+    cursor: pointer;
+    min-height: var(--mat-form-field-container-height);
+    transition: border-color 0.2s ease;
+
+    &:hover:not(.disabled) {
+      border-color: mat.m2-get-color-from-palette(gio.$mat-smoke-palette, darker80);
+    }
+
+    &.disabled {
+      opacity: 0.5;
```

```suggestion
      opacity: 0.4;
```
From memory, we have this value in the ui-particle components for this state.

---

Yes, it's not the most consistent between components in ui-particles, but 0.4 is used for a similar component to the new select-search component 👍